README.md

```diff
@@ -2,6 +2,15 @@
 
 This is the ACAP packaging of the Docker Engine to be run on Axis devices with container support.
 
+## Compatability
+
+The Docker ACAP is compatible with most ARTPEC-7 TPU and ARTPEC-8 cameras. It is recommended
```

Comment:
Perhaps a more general phrasing:

The Docker ACAP requires a container capable device. You may check the compatibility of your device model by running: